app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -4,15 +4,15 @@
 
 public class ParkingModel implements ParkingContract.Model {
 
-    private int parkingAvailable = -1;
+    private String parkingAvailable = "-1";
 
     @Override
-    public void setParkingAvailable(int parkingAvailable) {
+    public void setParkingAvailable(String parkingAvailable) {
```

Comment:
Why did you change the int value (that was correct) for an string?

Reply:
done